specs/protocol/isthmus/exec-engine.md

```diff
@@ -63,6 +84,33 @@ places a burden on users of the system in a post-fault-proofs world, where:
 Placing the [`L2ToL1MessagePasser`][l2-to-l1-mp] account storage root in the `withdrawalsRoot` field alleviates this burden
 for users and protocol participants alike, allowing them to propose and verify other proposals with lower operating costs.
 
+#### Genesis Block
+
+If isthmus is active at the genesis block, the withdrawals root is the empty withdrawals root, regardless of L2 state.
+
+#### State Processing
+
+At the time of state processing, the header for which transactions are being validated should not make it's `withdrawalsRoot`
+available to the EVM/application layer.
+
+#### P2P
+
+During sync, the block body withdrawal hash is computed from the withdrawals list in the block body. If Isthmus is
+active at such a block's header, we expect that the computed withdrawal hash must match the hash of an empty list of
+withdrawals. The header `withdrawalsRoot` MPT hash can be any non-null value in this case. Verification is performed
+after a header is available so that the header's timestamp can be checked to see if Isthmus is active.
+
+#### Backwards Compatibility Considerations
+
+Beginning at Shanghai and prior to Isthmus activation, the `withdrawalsRoot` field is set to the MPT root
```

Comment:
Which OP Stack is shanghai in? Could be better to use the OP Stack fork name

Reply:
Good point - Per [this issue](https://github.com/ethereum-optimism/optimism/issues/6728) and [blog post](https://blog.oplabs.co/canyon-hardfork/) , it is in `Canyon` hard fork. Modified to specify Canyon.